src/state/quick-open-mixin.ts

```diff
@@ -17,7 +17,7 @@ export default <T extends Constructor<HassElement>>(superClass: T) =>
       super.firstUpdated(changedProps);
 
       document.addEventListener("keydown", (e: KeyboardEvent) => {
-        if (e.code === "KeyP" && e.metaKey) {
+        if (this.isOSCtrlKey(e) && e.code === "KeyP") {
```

Comment:
We can just make it work with control on Mac too?
```suggestion
        if ((e.metaKey || e.ctrlKey) && e.code === "KeyP") {
```

Reply:
Oh, see your comments below